geom/src/polygon.rs

```diff
@@ -238,16 +238,21 @@ impl Polygon {
 
     /// Top-left at the origin. Doesn't take Distance, because this is usually pixels, actually.
     /// If radius is None, be as round as possible. Fails if the radius is too big.
```

Comment:
You can delete the first sentence here -- now the type name makes the fully round behavior obvious.

Reply:
done!